config/rbac/role.yaml

```diff
@@ -2,201 +2,31 @@
 apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
could you please restore this file and just add distributed resource there?

Reply:
Good idea, removed